audio.go

```diff
@@ -47,6 +40,29 @@ func (cmd MusicStop) Execute(app *App, done Promise) {
 	done.Complete()
 }
 
+// MusicPause is a command that will pause the music.
+type MusicPause struct{}
+
+func (cmd MusicPause) Execute(app *App, done Promise) {
+	app.pauseMusic()
+	done.Complete()
+}
+
+// MusicResume is a command that will resume the music.
+type MusicResume struct{}
+
+func (cmd MusicResume) Execute(app *App, done Promise) {
+	app.resumeMusic()
+	done.Complete()
+}
+
+// SetMusicVolume - Set volume for music (1.0 is max level)
+func (a *App) SetMusicVolume(volume float32) {
+	if a.isMusicReady() {
+		rl.SetMusicVolume(*a.music, volume)
+	}
+}
+
```

Comment:
I don't think we should export this function, as we do not expect it to be called out from the library. Volume will be handled from the settings menu of the application.

Reply:
Solved here: https://github.com/apoloval/pctk/pull/7